package.json

```diff
@@ -6,7 +6,6 @@
     "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
   },
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
```

Comment:
any reason u removed this?

Reply:
I don't recall removing it, I did not mess with package.json file tbh

Reply:
can u add that again @keshav-exe

Reply:
done